config/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/SIGBlockchain/project_aurum/internal/config"
+)
+
+func main() {
+	//open configuration file
+	configFile, err := os.Open("../cmd/config.json")
+	if err != nil {
+		log.Fatal("Failed to load configuration file : " + err.Error())
+	}
+	defer configFile.Close()
+
+	cfgData, err := ioutil.ReadAll(configFile)
+	if err != nil {
+		log.Fatal("Failed to read configuration file : " + err.Error())
+	}
+
+	cfg := config.Config{}
+	if err := json.Unmarshal(cfgData, &cfg); err != nil {
+		log.Fatal("Failed to unmarshall configuration data : " + err.Error())
+	}
+
+	//specify flags
+	versionU64 := flag.Uint64("version", uint64(cfg.Version), "enter version number")
+	flag.Uint64Var(&cfg.InitialAurumSupply, "initialaurumsupply", cfg.InitialAurumSupply, "enter a number for initial aurum supply")
```

Comment:
I think it would be easier on the user to make some of the long flags be single words.